QuaternionProprioState proprio  
AchievedGoal ag
ResetInfo resetState
sensor_msgs/Image shoulderImage
sensor_msgs/Image gripperImage



